src/mlpack/tests/loss_functions_test.cpp

```diff
@@ -711,4 +712,63 @@ BOOST_AUTO_TEST_CASE(MarginRankingLossTest)
       "-0.753830 1.336900 0.000000 0.000000 -0.207000 0.328810"), 1e-6);
 }
 
+/**
+ * Test for the multiLabel softmargin loss function, where loss is
+ * reduced to sum.
```

Comment:
Could you just keep the comment as `Simple test for the multiLabel softmargin loss function.` I know its a pick comment. Sorry :)

Reply:
Okay :)